test/imagemounter/list-developer-image-specs.js

```diff
@@ -0,0 +1,22 @@
+import chai from 'chai';
+import { findDeveloperImage } from '../../lib/imagemounter/utils/list_developer_image';
+import { fs } from '@appium/support';
+chai.should();
+describe('findDeveloperImage', function() {
+    it('should download and return the correct developer image for a given version', async function() {
+        const result = await findDeveloperImage('14.7.1', { githubRepo: 'JinjunHan/iOSDeviceSupport', subFolderList: ['iOSDeviceSupport'] });
+        result.developerImage.endsWith('/DeveloperDiskImage.dmg').should.be.true;
+        result.developerImageSignature.endsWith('/DeveloperDiskImage.dmg.signature').should.be.true;
+        (await fs.exists(result.developerImage)).should.be.true;
+        (await fs.exists(result.developerImageSignature)).should.be.true;
+    });
+
+    it('should throw an error if the developer image cannot be found', async function() {
+        try {
+            await findDeveloperImage('99.99.99', { githubRepo: 'JinjunHan/iOSDeviceSupport', subFolderList: ['iOSDeviceSupport'] });
```

Comment:
you could use .should.eventually.be.rejectedWith(/errMessage/)

Reply:
simply apply https://github.com/appium/appium-xcuitest-driver/blob/87839ac72393db1d79b9bf9a069fdcb890306161/test/functional/device/file-movement-e2e-specs.js#L10

Reply:
`.should.be.rejectedWith()`. you don't need `.eventually` unless you are making an assertion from `chai` proper or another plugin.  e.g., `.should.eventually.eql()`.